osu.Game/Online/API/APIDownloadRequest.cs

```diff
@@ -23,11 +26,11 @@ protected APIDownloadRequest()
 
         private void onSuccess()
```

Comment:
Remove method and do `base.Success += Success?.Invoke(filename);` in ctor instead?

Reply:
eh